infrastructure/crypto/Cargo.toml

```diff
@@ -1,5 +1,14 @@
 [package]
 name = "crypto"
 version = "0.0.1"
+edition = "2018"
 
 [dependencies]
+digest = "0.7.6"
+rand = "0.5.5"
```

Comment:
This should be in sync with the github.com/dalek-cryptography/ed25519-dalek dependency version

Reply:
We can do that, but It doesn't have to be. Cargo manages multiple dependencies seamlessly.

Reply:
Just checked-- they are aligned?

Reply:
See [this explanation of dependency versioning](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements)

For security reasons, we have a policy of **fixing** all dependencies to a specific version. Ahead of mainnet release, we will probably go even further and  fix versions to a specific git commit; but for now, while we're rapidly iterating,  using numeric versions is ok.

Reply:
Thank you for the link.
So the question boils down to why not make the text exactly the same as in the main ed25519-dalek library version instead of relying on the resolving mechanism. Thus to minimize the risk of us using different dependency versions in our derived code.